mne/viz/_brain/_brain.py

```diff
@@ -1261,7 +1261,8 @@ def _configure_tool_bar(self):
         self._renderer._tool_bar_add_file_button(
             name="movie",
             desc="Save movie...",
-            func=self.save_movie,
+            func=lambda: self.save_movie(
+                time_dilation=(1. / self.playback_speed)),
```

Comment:
I don't think this will do what we want. This will create a function with a *static* time dilation at the time of the toolbar button being added. So if you create a window, whatever the default playback speed is will be fixed here, and then no matter how you tweak the options in the interface it won't change.

What we probably need is a tiny `def _save_movie_clicked(...)` that calls `self.save_movie(..., time_dilation = 1. / self.playback_speed)`

Reply:
... another way around this would be to add the option `'widget'`, which means "use whatever value the interactive widget is set to". Then this call can be `lambda: self.save_movie(time_dilation='widget', ...)`

Reply:
It's not partial, you are sure the lambda won't be evaluated and then the updated value of `playback_speed` used?

Reply:
You're right, I was thinking about `partial` and passing kwargs, never mind!